.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: Linter check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+    - run: yarn install --frozen-lockfile
+    - run: yarn lint
+    
+  build:
+    name: Build check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+    - run: yarn install --frozen-lockfile
+    - run: yarn build
+    - run: yarn build:examples
+
+  test:
+    strategy:
+      matrix:
+        react-version: 
+          - ^16.8.0
```

Comment:
Let's leave off the caret on this, so that we actually test with 16.8.0, rather than the last 16.x version (16.14.0)

Reply:
I think there are not many changes between 16.14.0 and 17, but I'm not opposed to it.  We at least need 16.8.0, though.